api-spec/v1/openapi.yaml

```diff
@@ -60,6 +60,19 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/ProblemJson'
+  /commands/refund/redirect:
+    post:
+      tags:
+        - ecommerce
```

Comment:
this API will need to be revised, but for now can you provide a body with transactionId and pspId? and the response similar to the refund NPG body?